distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -887,7 +896,7 @@ public void testCachedContentLengthHeader() throws IOException {
                 .headers();
 
         //Get again to hit the cache
-        get(url).then()
+        get(url + "&mode=cache").then()
```

Comment:
Perhaps the itest doesn't have the same flow as a fully running DDF? I would guess the queries are formed differently. 

When I traced through CachingFederationStrategy during this test, both queries did not have a `QUERY_MODE` property set, so it looked like the cache wasn't being considered either time.